src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -243,6 +248,60 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 frontend_logger.log("Graph data file does not exist on the specified path", "error");
                 continue;
             }
+        } else if (line.compare(ADMDL) == 0) {
+            write(connFd, SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+            write(connFd, "\r\n", 2);
+
+            char graph_data[FRONTEND_DATA_LENGTH];
+            bzero(graph_data, FRONTEND_DATA_LENGTH + 1);
+            string name = "";
+            string path = "";
+
+            read(connFd, graph_data, FRONTEND_DATA_LENGTH);
+
+            std::time_t time = chrono::system_clock::to_time_t(chrono::system_clock::now());
+            string uploadStartTime = ctime(&time);
+            string gData(graph_data);
+
+            Utils utils;
+            gData = utils.trim_copy(gData, " \f\n\r\t\v");
+            frontend_logger.log("Data received: " + gData, "info");
+
+            std::vector<std::string> strArr = Utils::split(gData, '|');
+
+            if (strArr.size() < 2) {
+                frontend_logger.log("Message format not recognized", "error");
+                continue;
+            }
+
+            name = strArr[0];
+            path = strArr[1];
+
+
```

Comment:
There are some additional new lines which we do not need. Having one new line between two lines is sufficient enough.

Reply:
Fixed in 2d8c122

Reply:
Done